docs/evaluation/speech-audio.md

```
@@ -789,3 +789,43 @@ Output:
+Audio is downloaded automatically from HuggingFace. A single `prepare_data cs-fleurs` run produces all four test sets at once. Pass the parent group name (`cs-fleurs`), not the dotted subtask names. Use `--subsets` to prepare only some test sets.
+
+```bash
+ns prepare_data cs-fleurs \
+    --data_dir /path/to/data \
+    --cluster <cluster_name> \
+    --subsets read mms xtts-test1 xtts-test2
+```
+
+Output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)